plugins/modules/rabbitmq_user.py

```diff
@@ -106,6 +106,33 @@
     required: false
     default: on_create
     choices: ['on_create', 'always']
+  protocol:
+    description:
+      - Specify which TCP/IP protocol will be used
+    type: str
+    default: http
+    choices: ['http', 'https']
+  host:
+    description:
+      - Hostname from API
```

Comment:
```suggestion
      - Hostname from API.
```
I'm not sure, maybe to change `from` -> `of`?